app/models/profile.rb

```diff
@@ -0,0 +1,59 @@
+class Profile < ActiveRecord::Base
+  include Privacy
+
+  belongs_to :user
+
+  has_attached_file :curve, styles: {
```

Comment:
@ChaelKruip @grdw I'm guessing the new FinancialProfile model is intended to allow us to upload profiles to specify pricing in a market model?

Do we know yet what the contents of these profiles will look like? I ask because I'm pretty sure the demand/capacity scaling processors used on load curves won't apply to financial curves. I'm think we can move this to `LoadProfile` and add another `has_attached_file` to `FinancialProfile`? That's assuming Paperclip works with STI, but I don't see any reason it wouldn't...